Documentation/README_BoundingBox.md

```diff
@@ -64,3 +58,43 @@ Bounding box provides the following events. The example uses these events to pla
 - **Scale Ended**: Fires when scaling ends.
 
 <img src="../Documentation/Images/BoundingBox/MRTK_BoundingBox_Events.png" width="450">
+
+
+## Bounding box handle styles
+In default, when you just assign the [`BoundingBox.cs`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs) script, it will show the handle of the HoloLens 1st gen style. To use HoloLens 2 style handles, you need to assign proper handle prefabs and materials.
```

Comment:
```suggestion
By default, when you just assign the [`BoundingBox.cs`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs) script, it will show the handle of the HoloLens 1st gen style. To use HoloLens 2 style handles, you need to assign proper handle prefabs and materials.
```